src/components/utils/Input/InputControl.tsx

```diff
@@ -61,6 +61,10 @@ const InputControl: React.FC<InputControlProps> = ({
       }}
       _focusWithin={{
         borderColor: !invalid ? 'blue-400' : undefined,
+        '[data-prefix]': {
```

Comment:
can you not use  `span` or `& > span` here to avoid the `data-prefix` attribute?

Reply:
Done!